pipeline/terraform/test-environment/variables.tf

```
@@ -24,3 +24,16 @@ variable "mhs_log_level" {
   type = string
   description = "Log level for the MHS application."
 }
+variable "scr_log_level" {
+  type = "string"
```

V. minor: Don't know if it makes a difference, but:
```suggestion
  type = string
```